app/helpers/tournaments_helper.rb

```diff
@@ -1,2 +1,34 @@
 module TournamentsHelper
+
+    def success_per_question(results)
+        statistics = Array.new;
+        questions = Array.new;
+    
+        # initialize empty array
+        for index in 0..9 do
+          questions[index] = 0;
+        end
+    
+        results.each do |result|
+          # count all true questions
+          for index in 1..10 do
+            if (result.answers[index.to_s])
+              questions[index-1] = questions[index-1] + 1
+            end
+          end
+        end
+    
+        # create statistics object
+        for index in 1..10 do
+          singleQuestion = {
+            question: index.to_s,
+            percentage: (questions[index-1]/results.length()*100).to_s + "%"
+          }
+          statistics.append(singleQuestion)
+        end
+    
+        return statistics
```

Comment:
```suggestion
        statistics
```
It's the convention not to use a return statement if it's the last line of the function